packages/paste-website/src/pages/tokens/design-tokens-package.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Design Tokens Package
+package: '@twilio-paste/design-tokens'
+description: The Paste design tokens package allows you to install Paste design tokens in a variety of different formats
```

Comment:
Where is this description showing up?

Reply:
Nit, remote "Paste", sentence is too redundant.

Reply:
> Where is this description showing up?

Meta description, so Google and when you paste the url into Slack.